documents/Libraries/stdlib/sx-osl/mx_fractal3d_color4.osl

```diff
@@ -1,5 +1,5 @@
 void mx_fractal3d_color4(vector4 amplitude, int octaves, float lacunarity, float diminish, vector position, output color4 result)
 {
     color4 f = fBm(position, octaves, lacunarity, diminish, "snoise");
-    result = f * color4_pack(amplitude.x, amplitude.y, amplitude.z, amplitude.w);
+    result = f * color4(color(amplitude.x, amplitude.y, amplitude.z), amplitude.w);
```

Comment:
As far as I can see there is no color4_pack function anywhere. Just use default constructor instead.

Reply:
Isn't there a pack function (without color4_?

Reply:
I'll make the change.